plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-card {
+		padding: $large-gap;
+		border-radius: $grid-unit-05 !important;
+		margin-top: $large-gap;
+
+		&__content {
+			display: grid;
+			grid-template-columns: 1fr;
+			align-items: flex-start;
+			gap: $medium-gap;
+			justify-content: space-between;
+		}
+
+		&__header {
+			grid-row: auto;
+			grid-column: 1;
+			align-self: stretch;
+		}
+
+		&__icon {
+			width: $grid-unit-60;
+			height: $grid-unit-60;
+			flex-shrink: 0;
+			border-radius: $grid-unit;
+		}
+
+		&__details {
+			display: flex;
+			justify-content: flex-start;
+			align-items: center;
+			gap: $medium-gap;
+			background: $white;
+		}
+
+		&__meta {
+			display: flex;
+			flex-direction: column;
+			gap: 2px;
+			color: $gray-900;
+		}
+
+		&__title {
+			display: -webkit-box;
+			-webkit-box-orient: vertical;
+			-webkit-line-clamp: 2;
+			color: $gray-900;
+			font-size: $editor-font-size;
+			font-style: normal;
+			font-weight: 600;
+			line-height: $large-gap;
+			margin: 0;
+			padding: 0;
+			overflow: hidden;
+			text-overflow: ellipsis;
+		}
+
+		&__vendor {
+			display: flex;
+			gap: $grid-unit-05;
+			margin: 0;
+			padding: 0;
+		}
+
+		&__vendor a {
+			display: -webkit-box;
+			-webkit-box-orient: vertical;
+			-webkit-line-clamp: 1;
+			text-decoration: none;
+		}
+
+		&__description {
+			display: -webkit-box;
+			-webkit-box-orient: vertical;
+			-webkit-line-clamp: 3;
+			grid-row: auto;
+			grid-column: 1;
```

Comment:
Don't think these lines are doing anything (grid-row: auto is the default value, and grid-column: 1 is implied by the fact that there's only one column).

```suggestion
```